ByiringiroYves/MyDiaryP6
# Ch setup database 169524304

#### What does this PR do?
integrating postgesql and Nodejs(db connections)
#### Description of Task to be completed?
a server can be connected to a database
#### How should this be manually tested?
-clone my repository
-cd in it 
-start server by this command "npm start".
-make sure you have Postman and pg-Admin downloaded
#### What are the relevant pivotal tracker stories?
#169524304
